src/btree/bt_sync_obsolete.c

```diff
@@ -45,43 +46,53 @@ __sync_obsolete_inmem_evict(WT_SESSION_IMPL *session, WT_REF *ref)
     WT_TIME_AGGREGATE_INIT_MERGE(&newest_ta);
     do_visibility_check = obsolete = ovfl_items = false;
 
+    /*
+     *  The obsolete information on a page is identified as follows:
+     *  1. If the ref is obsolete using the newest stop time. In this case, mark it
+     *     for urgent eviction.
```

Comment:
Why it needs to be urgent?

Reply:
No reader is going to access these obsolete pages, if we can evict them ASAP, the cache space can be used for needed pages.